Realm/Realm/Helpers/SerializationHelper.cs

```diff
@@ -99,6 +100,9 @@ internal static void PreserveSerializers()
             _ = new BsonArraySerializer();
 
             _ = new ObjectSerializer();
+
+            _ = new EnumerableInterfaceImplementerSerializer<IList<object>, object>();
```

Comment:
Can this be `EnumerableInterfaceImplementerSerializer<IEnumerable<object>, object>` instead?

Reply:
I'll try it out and report back.

Reply:
You're correct. It works with IEnumerable as well.